File: Bus Management/DeleteBus/src/model/deleteModel.js
const connection = require('../config/db'); // Asegúrate de la ruta correcta

// Funciones CRUD usando async/await

const Bus = {
    // Registrar un bus
 deleteBus: async (id) => {
        try {
            const query = 'DELETE FROM buses WHERE id = ?';
            const [result] = await connection.query(query, [id]); // Uso de await
            return result;
        } catch (error) {
            console.error('Error to delete bus:', error);
            throw error;
        }
    }
};

module.exports = Bus;

File: UserManagement/DeleteUser/model/deleteModel.go
package model

import (
	"DeleteUser/config"
)

func DeleteUserByID(id string) error {
	db := config.GetDB()
	query := "DELETE FROM users WHERE id = $1"
	_, err := db.Exec(query, id)
	if err != nil {
		return err
	}
	return nil
}
